src/packages/core/src/servers/utils/http-response-codes.ts

```diff
@@ -1,9 +1,25 @@
+/**
+ * HTTP/1.1 Response Status-Codes, including the _required_ space character.
+ *
+ * e.g., `"200 "` or `"404 "`
+ *
+ * RFC Grammar:
+ *
+ * ```ebnf
+ * Status-Line = HTTP-Version SP Status-Code SP Reason-Phrase CRLF
+ * ```
+ *
+ * The Status-Codes defined here fullfill the `Status-Code SP` part of the above
+ * grammar.
+ *
+ * See https://datatracker.ietf.org/doc/html/rfc2616#section-6.1 for details.
+ */
 enum HttpResponseCodes {
-  OK = "200",
-  NO_CONTENT = "204",
-  BAD_REQUEST = "400",
-  NOT_FOUND = "404",
-  METHOD_NOT_ALLOWED = "405",
-  IM_A_TEAPOT = "418"
+  OK = "200 ",
+  NO_CONTENT = "204 ",
+  BAD_REQUEST = "400 ",
+  NOT_FOUND = "404 ",
+  METHOD_NOT_ALLOWED = "405 ",
+  IM_A_TEAPOT = "418 "
```

Comment:
Testing this is kind difficult. I _did_ manually test it by stepping through the lifecycle of a request sent to the Ganache server and the additional space character was present:

```
HTTP/1.1 200 <-- look! an extra space character!
Content-Type: application/json
Content-Length: 42

{"id":"1","jsonrpc":"2.0","result":"1234"}'
```

You can verify this by changing the following line in `./src/packages/core/node_modules/superagent/lib/node/index.js`:

```javascript
  this.req = mod.request(options);
```

with:

```javascript
  var req = this.req; // set tcp no delay
  req.on('socket', (socket) => {
    socket.on("data", data => {
console.log(`|${Buffer.from(data).toString("utf8").split("\r\n")[0]}|`);
    });
  });
```

and then running our `@ganache/core` tests.